lib/profile/common/add_row_button.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+import '../common/section.dart';
+import '../common/styles.dart';
+
+class AddRowButton extends StatelessWidget {
+  final String text;
+  final void Function() onTap;
+  
+  AddRowButton({Key key, @required this.text, @required this.onTap}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: onTap,
+      child: SectionRow(
+        children: <Widget> [
+          Container(
+            padding: EdgeInsets.only(right: 10.0),
+            child: addIcon
+          ),
+          Text(
+            text,
```

Comment:
I defs agree with being more specific in naming string text values but in this case, since it's a `text` is a prop/argument in a widget, i think it would look a little redundant when using the `AddRowButton` widget?
```
AddRowButton(
  addButtonText: addEducationRowText,
  onTap: () => handleAddTap(context)
)
```
as opposed to it just being:
```
AddRowButton(
  text: addEducationRowText,
  onTap: () => handleAddTap(context)
)
```

Reply:
Oh, gotcha. 👍